docker/faabric.dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM faasm/grpc-root:0.0.5
```

Comment:
This image will be rebuilt as part of the CI/ release process

Reply:
I am sorry about being so annoying with versions but I don't understand the policy used here.

Reply:
Unfortunately there isn't a policy at the moment, the different bits of the application have different versions and for now we'll just have to keep them in sync. Improving this is definitely a worthy task to add to the backlog.